lib/register_client.rb

```diff
@@ -143,13 +143,13 @@ def update_data_from_rsf(rsf, data_store)
           data_store.add_item(RegistersClient::Item.new(line))
         elsif command == 'append-entry'
           if params[1] == 'user'
-            @user_entry_number += 1
+            user_entry_number += 1
 
-            entry = Entry.new(line, @user_entry_number, params[1])
+            entry = Entry.new(line, user_entry_number, params[1])
           else
-            @system_entry_number += 1
+            system_entry_number += 1
 
-            entry = Entry.new(line, @system_entry_number, params[1])
+            entry = Entry.new(line, system_entry_number, params[1])
```

Comment:
Might be nice to assign these params to meaningful names, as it would make it easier to follow what the code is doing.

Reply:
Do you mean for `params[1]`? Perhaps, but that should be done in a separate PR, as there is a bunch of other cleanup that I'd also like to do. Let's keep this PR around the bug which is being fixed.

Reply:
Yes, right now we do it inconsistently e.g. we assign `param[0]` but not `param[1]`. But yes happy to pick this up in a separate PR.